### DISCUSSION TITLE
CSS not working

### DISCUSSION THREAD

**Discussion by ianckruger** [community/community#59669] [Programming Help]
### Body

Hey, I'm somewhat new to using github and have been trying to help fix a website. For some reason the CSS file of the website is not working, yet the html is being pushed through the repository just fine? I've checked through for any errors but I cant seem to find any. It seems like an issue with the publishing through github. Can anyone help? thanks.

this is the website:
https://github.com/duttamilan1/Magnolia

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by dumitrux** [comment]
## Issue
The browser doesn't understand SCSS directly, you need to compile the SCSS file into a regular CSS file before it can be used in your website.

You can test this by creating a simple CSS file and reference this file instead of `style.scss`.
In the `index.html` change the reference to `<link rel="stylesheet" href="assets/css/test.css"/>`.

## Solution
If you want to use SCSS, you should compile the SCSS file into CSS. 
Install the SCSS compiler as [Sass](https://sass-lang.com/install/) or [node-sass](https://github.com/sass/node-sass).

Once you have the SCSS compiler installed, you need to compile the `style.scss` file into CSS. For example, if you are using Sass, you can run the following Bash command:
```
sass assets/scss/style.scss assets/css/style.css
```
This command tells Sass to compile the `style.scss` file into a CSS file named `style.css` and save it in the `assets/css` directory.

After the compilation process completes successfully, you need to update the HTML file to reference the compiled CSS file instead of the SCSS file.
